CannonDriver.md

```diff
@@ -0,0 +1,297 @@
+# Cannon Driver
+
+[CANNON.js](http://schteppe.github.io/cannon.js/) is a native JavaScript physics library, which is one of the available choices of driver in aframe-physics-system
+
+This page describes how to use aframe-physics-system with the Cannon Driver
+
+## Installation
+
+### Scripts
+
+In the [dist/](https://github.com/c-frame/aframe-physics-system/tree/master/dist) folder, download the full or minified build. Include the script on your page, and all components are automatically registered for you:
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/c-frame/aframe-physics-system@v4.1.0/dist/aframe-physics-system.min.js"></script>
+```
+
+CDN builds for aframe-physics-system@v$npm_package_version:
+
+- [aframe-physics-system.js](https://cdn.jsdelivr.net/gh/c-frame/aframe-physics-system@v4.1.0/dist/aframe-physics-system.js) *(development)*
+- [aframe-physics-system.min.js](https://cdn.jsdelivr.net/gh/c-frame/aframe-physics-system@v4.1.0/dist/aframe-physics-system.min.js) *(production)*
+
+### npm
+
+```
+npm install --save aframe-physics-system
```

Comment:
this is out of date, not our version, right?

Reply:
reminder on this one

Reply:
npm install in general looks pretty flaky / in need of testing.  But I think that's separate from this PR so I'm going to leave for now.